src/types/ScholarshipAmount.test.ts

```diff
@@ -134,3 +150,110 @@ test('toString() - Range - unbounded', () => {
 
   expect(res).toBe('Varies');
 });
+
+test('toStorage() - Fixed', () => {
+  const amount = fixed(1000);
+  const res = toStorage(amount);
+
+  expect(res).toEqual({ type: AmountType.Fixed, min: 1000, max: 1000 });
+});
+
+test('toStorage() - Range - bounded', () => {
+  const amount = range(2, 20);
+  const res = toStorage(amount);
+
+  expect(res).toEqual({ type: AmountType.Varies, min: 2, max: 20 });
+});
+
+test('toStorage() - Range - no min', () => {
+  const amount = range(undefined, 20);
+  const res = toStorage(amount);
+
+  expect(res).toEqual({ type: AmountType.Varies, min: 0, max: 20 });
+});
+
+test('toStorage() - Range - no max', () => {
+  const amount = range(2, undefined);
+  const res = toStorage(amount);
+
+  expect(res).toEqual({ type: AmountType.Varies, min: 2, max: RANGE_MAX });
+});
+
+test('toStorage() - Range - unbounded', () => {
+  const amount = range(undefined, undefined);
+  const res = toStorage(amount);
+
+  expect(res).toEqual({
+    type: AmountType.Unknown,
+    min: UNKNOWN_MIN,
+    max: UNKNOWN_MAX,
+  });
+});
+
+test('toStorage() - FullTuition', () => {
+  const amount = fullTuition();
+  const res = toStorage(amount);
+
+  expect(res).toEqual({
+    type: AmountType.FullTuition,
+    min: FULL_TUITION,
+    max: FULL_TUITION,
+  });
+});
+
+test('toStorage() - Unknown', () => {
+  const amount = unknown();
+  const res = toStorage(amount);
+
+  expect(res).toEqual({
+    type: AmountType.Unknown,
+    min: UNKNOWN_MIN,
+    max: UNKNOWN_MAX,
+  });
+});
+
+test('fromStorage() - Fixed', () => {
+  const amount = fixed(1000);
+  const res = fromStorage(amount);
```

Comment:
For the `fromStorage()` tests you might want to do something like this instead to make sure you're converting from the storage represenation:
```suggestion
  const res = fromStorage(toStorage(amount));
```